src/components/PageLayout/PageFooter.vue

```diff
@@ -96,6 +96,8 @@
   }
 
   .language-bar {
+    margin-left: -22.5px;
```

Comment:
Is there a way we can do this without specifying exact numbers?

Reply:
Hm, I'd love to not hardcode in the margin here like this but I couldn't think of a clean approach that does so. 22.5 is the padding specified in EternaPage (line 82), and PageFooter & EPH don't have a reference to that value.

I imagine we could either extract the margin value of 22.5 to some global constant so they share a reference, or modify EternaPage to accept a parameter that tells it "set padding to 0" and then configure a separate padding in PageFooter/EPH so that the language bar/hero both overflow. Both approaches are somewhat more complicated than I thought warranted by the change, but happy to get your thoughts

Reply:
CSS variables?